extractor/filesystem/language/python/requirementstxt/extractor.go

```diff
@@ -0,0 +1,273 @@
+package requirementstxt
```

Comment:
Let's skip this from the PR for now, I'll try to consolidate the code here with our existing requirements.txt extractor in a separate PR.